### PR TITLE
fix: download vc button

### PR DESCRIPTION
**Pull request by huynguyen-hl**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR adds the ability to display the `Download` button, enabling users to download the issued VC after clicking the issue (Submit) button, provided the `includeDownload` setting is enabled (set to true). It ensures users can download the complete content stored in local storage by the `mergeToLocalStorage` service in the previous step.

Additionally, this PR includes documentation for the `getValueFromLocalStorage` function, detailing how it works and how to use it effectively.

## Related Tickets & Documents

https://github.com/gs-gs/fa-ag-trace/issues/727

## Mobile & Desktop Screenshots/Recordings

- Issue Transaction Event:

image

- Clicking the `Download` button:

image

- The downloaded content:

image

---

Docs site:

image
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📖 [Mock App docs site](https://uncefact.github.io/tests-untp/docs/mock-apps/)
- [ ] 📜 README.md
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by ashleythedeveloper** [comment]
@huynguyen-hl, can we add some documentation around component types (void, submit, etc..) and what that means? For example, if I have a void component before download, it won't download due to the previous component being void. It might also be beneficial to add a little something to the CustomButton component as well.

**Review by namhoang1604 — APPROVED** [review]

**Review by ldhyen99 — APPROVED** [review]

**Comment by huynguyen-hl** [comment]
> @huynguyen-hl, can we add some documentation around component types (void, submit, etc..) and what that means? For example, if I have a void component before download, it won't download due to the previous component being void. It might also be beneficial to add a little something to the CustomButton component as well.

@ashleythedeveloper, Yep, I agree with you. I’ll update all services to include the component types and provide a description of what they mean. Additionally, I’ll add more details to the CustomButton component documentation, explaining that if the last service in the services array is void, the button won’t be displayed because there is no data available for download.

**Review by namhoang1604 — DISMISSED** [review]

File: documentation/docs/mock-apps/services/get-value-from-local-storage.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 28
```
Comment (ldhyen99) [inline documentation/docs/mock-apps/services/get-value-from-local-storage.md:2]: Is it a duplicate? If it's not, skip this comment

image

**Review by ldhyen99 — APPROVED** [review]

File: documentation/docs/mock-apps/services/get-value-from-local-storage.md (hunk 1 above)
Comment (huynguyen-hl) [reply documentation/docs/mock-apps/services/get-value-from-local-storage.md:2]: Yep, it’s not duplicated in the `Services` tab. I think we only need to be concerned when using the same `sidebar_position` within the same parent tab.

**Pull request merged**